src/content/zh-CN/2020/capabilities.md

```diff
@@ -0,0 +1,332 @@
+---
+part_number: II
+chapter_number: 13
+title: Capabilities
```

Comment:
Could you also translate all of this metadata?

Reply:
translated